source/includes/_static_va_response_code.md

```diff
@@ -32,6 +32,7 @@ Response Code | Description
 202 | Request is Rejected (User ID is not active)
 202 | Request is Rejected (Multiple use VA is not allowed for use)
 202 | Request is Rejected (Custom VA is not allowed for use)
+990 | Request is Rejected (Required parameters is missing/invalid)
```

Comment:
<img width="770" alt="image" src="https://github.com/oyindonesia/external-api-docs/assets/5603786/812b3547-5231-4814-82fa-cb70bc0bbfd7">

bisa contoh ini aja ya, jadi dibikin per row. Yang di dalem kurung stringnya sesuai dengan error messagenya

Reply:
<img width="672" alt="image" src="https://github.com/oyindonesia/external-api-docs/assets/5603786/423bb501-ae29-4c65-8b0d-a2a7c9715326">
214 dan 219

Reply:
Isee, jadi lebih baik dimention langsung nama param-nya di dalam kurungnya kah, instead of dibikin generic `invalid param`?